src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig

```diff
@@ -57,14 +54,6 @@
 {% endblock %}
 
 {% block panelContent %}
-    {% set filter = request.query.get('state', '-1') %}
```

Comment:
It's a mistake. I'm sorry. If this proposal is approved I'll review each panel very carefully. Thanks.